.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 default_language_version:
   # force all unspecified python hooks to run python3
-  python: python3
+  python: python2
```

Comment:
Is `python2` available on all systems by now? I thought `python` was supposed to stay Python 2 forever.

Reply:
Seems to be available on Ubuntu at least.